### PR TITLE
peng/update ui designer

### PR DESCRIPTION
**Pull request by nylira**
- [x] rename the ui designer / developer job to user interface designer
- [x] don't require the designer to code
- [x] expand upon the job requirements
- [x] use the new 'design' career section

I changed the UI Designer/Developer role to just "User Interface Designer". Here are my thoughts. 

We want to hire a good UI designer, and a good one who can code is extremely difficult to find. In a small company, a UI designer/developer combo is very handy, but as we expand, it's entirely okay to look for people who are more specialized. 

One of our goals is have a solid Cosmos Style Guide for our developers to use as an implementation tool for new components. But this won't be created as long as we're designing everything in code. Our CSS could use a lot of refactoring because we've been using it as a design tool, without caring about redundant code (@faboweb mentions this on occasion during new sprints). All of our component design work should be moved to Sketch, and the person we hire for this role could help with that.

And as we start thinking about building interfaces for mobile, it occurs to me that finding a designer / developer who can write front end code for web and mobile is asking for too much in one person. Let's reduce this role's responsibilities so we can find a stronger candidate, sooner.

### CODE REVIEW

File: md/careers/design-ui-designer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# User Interface Designer
+
+### This is a contract position located in either Toronto, Berlin, or San Francisco with the possibility of long term employment.
+
+### Role summary:
+
+Tendermint is seeking an exceptional user interface designer to join the burgeoning Tendermint Design team. The designer will be responsible for researching, designing, and prototyping new user experiences based on product requirements. Deliverables include design mockups, prototyping, production of visual assets, and documentation required for proper implementation. The candidate must flourish in a fast-paced environment at a blockchain startup.
+
+This position will work closely with our Head of Design and Voyager team to design user experiences for our websites and software releases. The applicant should be a solid communicator and visual problem solver who has experience working with distributed teams.
```
Comment (jolesbi) [inline md/careers/design-ui-designer.md]: "Head of Design and _the_ Voyager team"

File: md/careers/design-ui-designer.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# User Interface Designer
```
Comment (jolesbi) [inline md/careers/design-ui-designer.md]: this is great peng. makes a lot of sense to me.

File: md/careers/design-ui-designer.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### This is a contract position located in either Toronto, Berlin, or San Francisco with the possibility of long term employment.
```
Comment (jolesbi) [inline md/careers/design-ui-designer.md]: contract as opposed to full time?

File: md/careers/design-ui-designer.md (hunk 2 above)
Comment (jolesbi) [reply md/careers/design-ui-designer.md]: Senior User Interface Designer? Senior Product Designer?

File: md/careers/design-ui-designer.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+* 5+ years of web and mobile interface design experience
+* Strong command of interaction and user-centered design principles
+* Passion about graphic design, color usage, and subtle details
+* Clear and concise communication skills
+* The ability to create work independently, as well as present actionable solutions to the Design and Voyager teams
+* Experience working with developers in both scrum and agile methodologies
+* Demonstrated mastery of Sketch, Figma, or other visual design tools
+* Interest in emerging interactive technology in web, mobile, and other devices
+
+### The ideal candidate has:
```
Comment (jolesbi) [inline md/careers/design-ui-designer.md]: i think the primary responsibilities section is critically important.

File: md/careers/design-ui-designer.md (hunk 2 above)
Comment (nylira) [reply md/careers/design-ui-designer.md]: Senior User Interface Designer!

File: md/careers/design-ui-designer.md (hunk 4 above)
Comment (nylira) [reply md/careers/design-ui-designer.md]: It's been added.

Comment (jolesbi) [reply md/careers/design-ui-designer.md]: do you think it is redundant to have both `We're looking for someone who has:` and `The ideal candidate has:`?

Comment (nylira) [reply md/careers/design-ui-designer.md]: @jolesbi yeah, it's been resolved

**Pull request merged**